src/test/crux-gateway/asymmetric-encryption.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai';
+import 'mocha';
+import { BasicKeyManager } from '../../infrastructure/implementations/basic-key-manager';
+import { EncryptionManager } from '../../core/domain-services/crux-messenger';
+
+describe("Asymmetric encryption tests", () => {
+    let basicKeyManager: BasicKeyManager;
+    const testPrivateKey = "2982735d0b69751e1d13fcb045757e372c1d85b8bdc66995a5a073be648e5f26";
+    const testPublicKey = "03da5e78c86e06c0b61580d73a38e31a511f36f1a5238a144b98b18608b90fa72e";
+    const sampleMessage = "exampleValue";
+    describe("Asymmetric encryption using eccrypto tests", () => {
```

Comment:
Lets add separate test cases for `encrypt` and `decrypt` as well

Reply:
Done